api/config.js

```diff
@@ -20,20 +23,26 @@ if (!isProduction && !isDevelopment) {
 
 if (!secret) {
 	if (isProduction) {
-		logger.error("the secret was not provided");
 		logger.error("set HACKILLINOIS_SECRET to a secure, random string");
-
 		process.exit(1);
 	}
 
 	secret = 'NONE';
 }
 
+if (!mailApiKey) {
+	if (isProduction) {
+		logger.error("set HACKILLINOIS_MAIL_KEY to the API key");
```

Comment:
More specific error message, something like "set ENV variable HACKILLINOIS_MAIL_KEY to API key provided by mail service provider"